catalyst/contrib/nn/modules/__init__.py

```diff
@@ -31,3 +31,7 @@
     scSE,
     cSE,
 )
+
+from catalyst.contrib.nn.modules.softmax import SoftMax
+from catalyst.contrib.nn.modules.arcface import ArcFace
+from catalyst.contrib.nn.modules.cosface import CosFace
```

Comment:
as far as this is criterions, could we move them to `contrib/nn/criterion`? what do you think about this?